packages/stripe/src/Utils.js

```diff
@@ -0,0 +1,43 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  logResponse,
+} from '@openfn/language-common/util';
+import nodepath from 'node:path';
+
+export const prepareNextState = (state, response) => {
+  const { body, ...responseWithoutBody } = response;
+
+  if (!state.references) {
+    state.references = [];
+  }
+
+  return {
+    ...composeNextState(state, body),
+    response: responseWithoutBody,
+  };
+};
+
+export const request = (configuration = {}, method, path, options) => {
+  const { baseUrl, apiKey, apiVersion = 'v1' } = configuration;
```

Comment:
Can we also default `baseUrl` to `https://api.stripe.com`. Or is there a good reason we shouldn't do that ?